libs/utils/src/lib/gdpr-cookie/cookie-banner/cookie-banner.component.scss

```
@@ -1,6 +1,51 @@
 :host {
   position: fixed;
   bottom: 0;
+  right: 20px;
   z-index: 2;
-  width: 100%;
+  width: 500px;
+
+  a {
+    position: absolute;
+    top: 16px;
+    right: 16px;
+  }
+
+  img {
+    width: 50px;
+  }
+
+  main, form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    div {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      gap: 16px; 
+
+      mat-checkbox {
+        margin-left: auto;
+      }
+
+      mat-icon {
+        width: 16px;
+      }
+    }
+  }
+
+  main {
+    gap: 16px;
+  }
+
+  section {
+   margin-top: 16px;
+  }
+
+  .mat-flat-button {
+    margin-right: 8px;
+  }
```

```suggestion
  section {
     margin-top: 16px;
     display: flex;
     gap: 8px;
  }
```
How I would have done it. In my opinion more robust against small changes (e.g. button style or button order)